www/Media.js

```diff
@@ -156,7 +156,10 @@ Media.prototype.resumeRecord = function() {
  * Release the resources.
  */
 Media.prototype.release = function() {
-    exec(null, this.errorCallback, "Media", "release", [this.id]);
+    var me = this;
+    exec(function() {
+      delete mediaObjects[me.id];
```

Comment:
We should probably use splice instead - delete won't change the length or reindexes the array.

Reply:
`mediaObjects` is an object, not an array. It is defined on line 26.

Reply:
Oh indeed, sorry!